src/tracking.ts

```diff
@@ -89,9 +86,6 @@ export const analyzeQueries = (queries: KustoQuery[], datasourceSrv: DataSourceS
       datasources[JSON.stringify(query.datasource)] = dsSettings;
     }
     if (dsSettings) {
-      if (dsSettings.jsonData?.onBehalfOf) {
-        counters.on_behalf_of_queries++;
-      }
```

Comment:
this information is still useful, you can check if the `authType` is  `clientsecret-obo` instead.

Reply:
Tracking was restored.